### DISCUSSION TITLE
Where can I download PCbox emulator?

### DISCUSSION THREAD

**Discussion by unclecaptain5426** [PCBox/PCBox#75] [Q&A] [answered]
I tried looking and downloading for PCbox emulator, I googled everywhere, but without luck, and so, I can't find it?
Any ideas?

**Comment by TechGuyC** [comment]
you need to click on actions,then click on MSYS2 Makefile then click on the latest version with the green symbol,the ones with red symbol does not work

**Reply by bachimusprime** [reply, marked as answer]
Actually, CMake builds are the recommended ones now. Also, you NEED to be logged into GitHub, and you also need to scroll down to Artifacts to find the builds. There are detailed instructions on the Discord in the pinned messages in the #general channel.
